### PR TITLE
Example Code

### PR DESCRIPTION
**Pull request by ah1053**
**Example Code** 
1. WebSocket Example
2. Rest Example (UnityWebRequest - HttpClient)
3. Deleting Example.cs (used to have the two examples above)

### CODE REVIEW

File: Runtime/Scripts/ExampleRest.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            }
+            catch (DmeDnsException)
+            {
+                mxi.UseWifiOnly(true);
+                await mxi.RegisterAndFindCloudlet();
+            }
+            mxi.GetAppPort(LProto.L_PROTO_HTTP);
+            string url = mxi.GetUrl("http");
+            Debug.Log("Rest URL is : " + url);
+
```
Comment (lgarner) [inline Runtime/Scripts/ExampleRest.cs:37]: // Tiny comment about: do something with the new edge server URL "here"

File: Runtime/Scripts/ExampleRest.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            StartCoroutine(RestExample(url)); // You can use RestExampleHttpClient
```
Comment (lgarner) [inline Runtime/Scripts/ExampleRest.cs:38]: "You can instead use..."

File: Runtime/Scripts/ExampleRest.cs (hunk 1 above)
Comment (ah1053) [reply Runtime/Scripts/ExampleRest.cs:37]: done

File: Runtime/Scripts/ExampleRest.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            }
+            catch (DmeDnsException)
+            {
+                mxi.UseWifiOnly(true);
+                await mxi.RegisterAndFindCloudlet();
+            }
+            mxi.GetAppPort(LProto.L_PROTO_HTTP);
+            string url = mxi.GetUrl("http");
+            Debug.Log("Rest URL is : " + url); // Once you have your edge server url you can start communicating with your Edge sever deployed on MobiledgeX Console
+
```
Comment (brucearmstrong) [inline Runtime/Scripts/ExampleRest.cs:37]: nit: sever->server

**Pull request merged**